Library/Homebrew/os/linux.rb

```diff
@@ -47,6 +47,10 @@ def full_version
       Version::NULL
     end
 
+    def outdated_release?
```

Comment:
Ideally let's fix the conditional so this isn't needed. Adjust the logic accordingly to fix.

Reply:
My other idea was to define a constant `{HOMEBREW_}EARLIEST_SUPPORTED_MACOS` somewhere and use that, but I wasn't sure where to put it. Also in `formula_installer`?

Reply:
Since we have `(!OS.mac? || !OS::Mac.outdated_release?)` (unless you are planning on changing that?) then this shouldn't be needed anymore.

Reply:
I thought that too, but I think shellcheck complained about it. Maybe I was confused. Let me try again.

Reply:
Ok, I was confused. Thanks for the nudge.